tests/inputFields.spec.ts

```diff
@@ -5,91 +5,84 @@ test.beforeEach( async({page}) => {
   //1. Select the PET TYPES menu item in the navigation bar
   await page.getByText('Pet Types').click()
   //2. Add assertion of the "Pet Types" text displayed above the table with the list of pet types
-  expect(await page.locator('h2').getByText('Pet Types').isVisible()).toBeTruthy()
+  expect(await page.locator('h2').getByText('Pet Types').textContent()).toEqual('Pet Types')
```

Comment:
Little bit better but not perfect.
Here is what will be your error message when text Pet Types is not displayed:
![image](https://github.com/b-w-cole/playwright-homework/assets/21285622/d4bb3b90-e023-4d3b-9e05-e073c2614896)
You can see, that assertion is not failed. You have test failure because locator is not found. And it happened, because expected text is part of your locator. So in the assertion like this - assertion will never happen, because locator will fail first.
When you asserting text - this text should not be part of your locator. 

Also, it always better to use a locator type of the assertions, because it has own timeout of 5 seconds. In this example, application was waiting to fail up to the test timeout, waiting for the locator